projects/plugins/social/src/js/editor.js

```diff
@@ -105,6 +106,7 @@ const JetpackSocialSidebar = () => {
 				<PublicizePanel>
 					<PanelDescription />
 				</PublicizePanel>
+				<SocialImageGeneratorPanel />
```

Comment:
Is there a feature check here that will prevent this from rendering without the plan/feature flag?

Reply:
Yeah, I'll add it after we merge it here: #29001

Reply:
Nice!